index.html

```diff
@@ -1,180 +1,371 @@
-      </ul>
-    <button class="project-btn flex">
-      <span>See this project</span>
-      <img src="images/icons/ic_arrow_right.svg" width="20px" height="15px" alt="#" srcset="">    
-    </button>
-</article>
+    <!-- about me section -->
+    <section class="about flex">
+      <div id="about-me">
+        <h3 class="title popins">About</h3>
+        <p id="about-me-details" class="popins">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
+        <button id="get-cv-btn" class="btn flex popins">
+          <span>Get my CV</span
+          ><img
+            src="Images/icons/ic_download.svg"
+            width="24"
+            height="24"
+            alt=""
+          />
+        </button>
+      </div>
+      <div class="details-container popins">
+        <ul>
+          <li id="langs" class="card">
+            <h5>Languages</h5>
+            <ul class="list-i">
+              <li>JavaScript (ES5/ES6)</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+          </li>
+          <li id="frameworks" class="card">
+            <h5>Frameworks</h5>
+            <ul class="list-i">
+              <li>React.js</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>Capibara</li>
+            </ul>
+          </li>
+          <li id="skills" class="card">
+            <h5>Skills</h5>
+            <ul class="list-i">
+              <li>Database Management</li>
+              <li>CLI</li>
+              <li>API Design</li>
+              <li>Version Control</li>
+              <li>Web Development</li>
+            </ul>
+          </li>
+          <li id="about-me-bg" class="card"></li>
+        </ul>
+      </div>
+    </section>
+
+    <!-- Contact ME -->
 
-<article class="project flex">
-  <img src="images/icons/Project-5.svg" alt="Project Image4" class="project-img">
-    <h3>Project name goes here</h3>
-    <ul class="project-languages flex">
-      <li class="project-language">HTML/CSS</li>
-      <li class="project-language">Ruby on Rails</li>
-      <li class="project-language">JavaScript</li>
-    </ul>
-  <button class="project-btn flex">
-    <span>See this project</span>
-    <img src="images/icons/ic_arrow_right.svg" width="20px" height="15px" alt="#" srcset="">     
-  </button>
-</article>
+    <section id="contact-me" class="playfair">
+      <img
+        src="images/icons/image_geometry_1.svg"
+        alt=""
+        class="contact-t-img"
+      />
+      <div id="contact-box" class="flex">
+        <h3 id="get-started" class="title">Get started</h3>
+        <p class="contact-intro">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
```

Comment:
- [ ] As I mentioned above :point_up_2:  please stick to the Figma design your contact form is not the same as the Figma design so kindly fix this issue :+1: check below :point_down:  the comparison between your design and Figma design :slightly_smiling_face: 

**Your Design**
![screencapture-127-0-0-1-5500-2023-06-17-00_47_42 (1)](https://github.com/Saritalitu23/Microverse-Portfolio-Project-/assets/45226848/4cab528d-8d8e-40b5-9b5f-7b07264188a3)

**Figma Design**
![Screenshot from 2023-06-17 00-46-28](https://github.com/Saritalitu23/Microverse-Portfolio-Project-/assets/45226848/84619f05-3875-4be1-af4a-8f981adbad4c)

- [ ] If you check your text area height it's very huge kindly reduce the height of the text area :+1: 
- [ ] Please fix the path of your images it's broken so kindly fix this issue as well :+1: 
- [ ] Please use the same `font styling` and spacing mentioned in the Figma design :+1:

Reply:
> * [ ]  As I mentioned above point_up_2  please stick to the Figma design your contact form is not the same as the Figma design so kindly fix this issue +1 check below point_down  the comparison between your design and Figma design slightly_smiling_face
> 
> **Your Design** ![screencapture-127-0-0-1-5500-2023-06-17-00_47_42 (1)](https://user-images.githubusercontent.com/45226848/246524948-4cab528d-8d8e-40b5-9b5f-7b07264188a3.png)
> 
> **Figma Design** ![Screenshot from 2023-06-17 00-46-28](https://user-images.githubusercontent.com/45226848/246524993-84619f05-3875-4be1-af4a-8f981adbad4c.png)
> 
> * [ ]  If you check your text area height it's very huge kindly reduce the height of the text area +1
> * [ ]  Please fix the path of your images it's broken so kindly fix this issue as well +1
> * [ ]  Please use the same `font styling` and spacing mentioned in the Figma design +1

@Saritalitu23, the same thing apply here. Kindly look into them and fix.

### Your Design
[your-app-desktop.webm](https://github.com/Saritalitu23/Microverse-Portfolio-Project-/assets/26597432/3325cbaa-ec7b-4f59-a9f1-e07e22dcec53)